lib/plugins/aws/package/compile/layers/index.test.js

```diff
@@ -223,17 +223,21 @@ describe('AwsCompileLayers', () => {
       return expect(
         Promise.all([awsCompileLayers.compileLayers(), secondAwsCompileLayers.compileLayers()])
       ).to.be.fulfilled.then(() => {
+        const resources =
+          awsCompileLayers.serverless.service.provider.compiledCloudFormationTemplate.Resources;
+        const secondResources =
+          secondAwsCompileLayers.serverless.service.provider.compiledCloudFormationTemplate
+            .Resources;
         expect(
-          _.findKey(
-            awsCompileLayers.serverless.service.provider.compiledCloudFormationTemplate.Resources,
-            r => r.Type === 'AWS::Lambda::LayerVersion'
-          )
+          Object.keys(resources).find(key => {
+            const r = resources[key];
+            return r.Type === 'AWS::Lambda::LayerVersion';
```

Comment:
Le'ts just write it as `return resources[key].Type === 'AWS::Lambda::LayerVersion'`